src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -38,9 +38,16 @@ namespace picongpu
          */
         HINLINE float3_X laserLongitudinal( uint32_t currentStep, float_X& phase )
         {
-            const float_64 runTime = DELTA_T*currentStep;
+            /* initialize the laser not in the first cell is equal to a negative shift
+             * in time
+             */
+            constexpr float_X laserTimeShift = laser::laserInitPlaneY * CELL_HEIGHT / SPEED_OF_LIGHT;
+            const float_64 runTime = DELTA_T * currentStep - laserTimeShift;
             const float_64 f = SPEED_OF_LIGHT / WAVE_LENGTH;
 
+            /* calculate focus position relative to the laser initialization plane */
+            const float_X focusPos = FOCUS_POS - laser::laserInitPlaneY * CELL_HEIGHT / SPEED_OF_LIGHT;
```

Comment:
ERROR:
`focusPos` is a position in space but `laser::laserInitPlaneY * CELL_HEIGHT / SPEED_OF_LIGHT` is a time.
Please fix it to 
```c++
const float_X focusPos = FOCUS_POS - laser::laserInitPlaneY * CELL_HEIGHT; 
```